AK/Utf8View.cpp

```diff
@@ -212,6 +226,11 @@ size_t Utf8CodepointIterator::code_point_length_in_bytes() const
     size_t code_point_length_in_bytes = 0;
     u32 value;
     bool first_byte_makes_sense = decode_first_byte(*m_ptr, code_point_length_in_bytes, value);
+
+    if (!first_byte_makes_sense) {
+        // We will output a replacement character for this byte and treat it as a codepoint of size 1.
+        return 1;
```

Comment:
We have a problem here:
Does this function return the length of the data as stored inside the view or as it will be returned by `operator*`?
If the former, returning 1 is correct. If the latter, we need to return 3 (as U+FFFD is encoded as `0xef 0xbf 0xbd`). See the review comment for more thoughts about this.

Reply:
I don't think it's as much of an issue as you make it seem (reasons explained below), but I agree that it could be clearer. I updated the name of the method from `code_point_length_in_bytes()` to `underlying_code_point_length_in_bytes()` and added a comment in the header explaining this behavior.